src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -69,66 +39,71 @@ BaseCouplingScheme::BaseCouplingScheme(
       _m2n(m2n),
       _maxTime(maxTime),
       _maxTimeWindows(maxTimeWindows),
-      _timeWindows(1),
-      _timeWindowSize(timeWindowSize),
-      _maxIterations(maxIterations),
       _iterations(1),
-      _totalIterations(1),
       _totalIterationsCoarseOptimization(1),
+      _maxIterations(maxIterations),
+      _couplingMode(cplMode),
+      _totalIterations(1),
+      _timeWindows(1),
+      _timeWindowSize(timeWindowSize),
       _validDigits(validDigits)
 {
   PRECICE_CHECK(not((maxTime != UNDEFINED_TIME) && (maxTime < 0.0)),
-                "Maximum time has to be larger than zero!");
+                "Maximum time has to be larger than zero.");
   PRECICE_CHECK(not((maxTimeWindows != UNDEFINED_TIME_WINDOWS) && (maxTimeWindows < 0)),
-                "Maximum number of time windows has to be larger than zero!");
+                "Maximum number of time windows has to be larger than zero.");
   PRECICE_CHECK(not((timeWindowSize != UNDEFINED_TIME_WINDOW_SIZE) && (timeWindowSize < 0.0)),
-                "Time window size has to be larger than zero!");
+                "Time window size has to be larger than zero.");
   PRECICE_CHECK((_validDigits >= 1) && (_validDigits < 17),
-                "Valid digits of timestep length has to be between 1 and 16!");
+                "Valid digits of time window size has to be between 1 and 16.");  // TODO really time window size?
```

Comment:
```suggestion
                "Valid digits of time window size has to be between 1 and 16.");
```
Yes, this is how the configuration defines it.